src/OpenLoco/src/World/CompanyAi/CompanyAi.cpp

```diff
@@ -3263,6 +3263,10 @@ namespace OpenLoco
         auto maxBaseZ = std::numeric_limits<SmallZ>::min();
         for (auto& aiStation : thought.stations)
         {
+            if (!World::validCoords(aiStation.pos))
```

Comment:
Any idea how these coords get to be invalid in the first place? Just worried this might be hiding a bug.

Reply:
This is how vanilla handles it I think if you follow the logic you can see why it might have invalid positions for stations that aren't in use.